src/docs/product/accounts/sso/index.mdx

```diff
@@ -131,3 +131,7 @@ When prompted with the Rippling Metadata URL, copy this into the Sentry Rippling
 #### Custom SAML2 Integration
 
 For other SAML2 SSO providers not listed above, Sentry provides generic connectors for [SAML2 based authentication](/product/accounts/sso/saml2/), which may be configured manually.
+
+## SCIM Provisioning
+System for Cross-Domain Identity Management ([SCIM](http://www.simplecloud.info/)) is a standard implemented by Identity Providers and applications in order to facilitate automated identity management. Sentry supports a subset of the specification for provisioning organization members and teams. See the relevant documentation for your use case:
```

Comment:
```suggestion
System for Cross-Domain Identity Management ([SCIM](http://www.simplecloud.info/)) is a standard implemented by Identity Providers and applications to facilitate automated identity management. Sentry supports a subset of the specification for provisioning organization members and teams. See the relevant documentation for your use case:
```

Reply:
I think this content would be better placed above the Providers listing. In addition, does the content provided [here](https://sentry-docs-7c0f13y4f.sentry.dev/product/accounts/sso/#okta) need refreshing?

Reply:
I think it makes sense to have it below Providers, as the SSO setup is a prerequisite for SCIM. Let me know if that makes sense. And SCIM doesn't change the flow for setting up SSO (it's like an add-on for more functionality basically), so i think keeping the content the same there made sense to me.